src/root_terraform/main.tf

```diff
@@ -1,9 +1,19 @@
-module ibm_cis_integration {
+module ibm_cis_ce_integration {
     source = "../ce/terraform"
+    ibmcloud_api_key = var.ibmcloud_api_key
     cis_name = var.cis_name
     resource_group = var.resource_group
     app_url = var.app_url
     cis_domain = var.cis_domain
-    ibmcloud_api_key = var.ibmcloud_api_key
     pool_name = var.pool_name
+}
+
+module ibm_cis_iks_integration {
+    source = "../iks/terraform"
+    ibmcloud_api_key = var.ibmcloud_api_key
+    cis_name = var.cis_name
+    resource_group = var.resource_group
+    cis_domain = var.cis_domain
+    cluster_id = var.cluster_id
+    token = var.token
```

Comment:
var.token is not needed anymore

Reply:
Alright, I accidentally worked off the wrong branch so once PR #10 is merged, I can pull down that code and then rework the module based off of that.